config/s2i/gold/jenkins-persistent-buildconfig-template.yaml

```diff
@@ -0,0 +1,275 @@
+                -Djenkins.model.Jenkins.logStartupPerformance=true
+                -Djava.security.egd=file:/dev/./urandom
+                -Djenkins.install.runSetupWizard=false
+                -Dhudson.DNSMultiCast.disabled=true
+                -Dhudson.model.ParametersAction.keepUndefinedParameters=true
+          - name: JAVA_GC_OPTS
+            value: >-
+                -XX:+UseParallelGC -XX:MaxPermSize=100m -XX:MinHeapFreeRatio=20
+                -XX:MaxHeapFreeRatio=40 -XX:GCTimeRatio=4
+                -XX:AdaptiveSizePolicyWeight=90 -XX:MaxMetaspaceSize=500m
+          - name: GIT_SSL_NO_VERIFY
+            value: 'true'
+          resources:
+            limits:
+              memory: "${MEMORY_LIMIT}"
+          volumeMounts:
+          - name: "${JENKINS_SERVICE_NAME}-data"
+            mountPath: "/var/lib/jenkins"
+          terminationMessagePath: "/dev/termination-log"
+          imagePullPolicy: IfNotPresent
+          capabilities: {}
+          securityContext:
+            capabilities: {}
+            privileged: false
+        volumes:
+        - name: "${JENKINS_SERVICE_NAME}-data"
+          persistentVolumeClaim:
+            claimName: "${JENKINS_SERVICE_NAME}"
+        restartPolicy: Always
+        dnsPolicy: ClusterFirst
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: "${JENKINS_SERVICE_NAME}"
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirectreference.jenkins: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"${JENKINS_SERVICE_NAME}"}}'
+- kind: RoleBinding
+  apiVersion: v1
+  metadata:
+    name: "${JENKINS_SERVICE_NAME}_edit"
+  groupNames:
+  subjects:
+  - kind: ServiceAccount
+    name: "${JENKINS_SERVICE_NAME}"
+  roleRef:
+    name: edit
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: "${JNLP_SERVICE_NAME}"
+  spec:
+    ports:
+    - name: agent
+      protocol: TCP
+      port: 50000
+      targetPort: 50000
+      nodePort: 0
+    selector:
+      name: "${JENKINS_SERVICE_NAME}"
+    type: ClusterIP
+    sessionAffinity: None
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: "${JENKINS_SERVICE_NAME}"
+    annotations:
+      service.alpha.openshift.io/dependencies: '[{"name": "${JNLP_SERVICE_NAME}",
+        "namespace": "", "kind": "Service"}]'
+      service.openshift.io/infrastructure: 'true'
+    creationTimestamp:
+  spec:
+    ports:
+    - name: web
+      protocol: TCP
+      port: 80
+      targetPort: 8080
+      nodePort: 0
+    selector:
+      name: "${JENKINS_SERVICE_NAME}"
+    type: ClusterIP
+    sessionAffinity: None
+parameters:
+- description: Git source URI for Jenkins S2I
+  name: REPO_URL
+  required: true
+  value: https://github.com/CentOS-PaaS-SIG/ci-pipeline.git
+- description: Git branch/tag reference
+  name: REPO_REF
+  value: master
+- description: Path within Git project to build; empty for root project directory.
+  name: CONTEXT_DIR
+  value: config/s2i/jenkins/master
```

Comment:
Is this the correct path?  Wouldn't this be the path to config/s2i/gold/jenkins/master ?

Reply:
Yes, thank you

Reply:
+1